src/couch_log/src/couch_log_formatter.erl

```diff
@@ -58,18 +58,18 @@ format(Level, Pid, Msg) ->
 
 format({error, _GL, {Pid, "** Generic server " ++ _, Args}}) ->
```

Comment:
I also think it'd be a good idea to have this function be do_format, and then we can wrap it in something ilke:

```erlang
format(Arg) ->
    try
        do_format(Arg)
    catch T:R ->
        log_that_it_bew_up(T, R)
    end.
```